common/core/gpudevice.cpp

```diff
@@ -649,7 +649,9 @@ void GpuDevice::HandleRoutine() {
       ITRACE("Successfully grabbed the hwc lock.");
     }
 
+	ETRACE("hwc before  set master.");
```

Comment:
use ITRACE instead of ETRACE if it is not error message

Reply:
Suggest just to remove it

Reply:
since we are do the seamless translate with other team, this log can provide the time which we get the master. so we can keep it for a short time, and remove it later, once the task force done.